ci-operator/jobs/openshift/release/openshift-release-infra-periodics.yaml

```diff
@@ -3015,3 +3015,117 @@ periodics:
     - name: manifest-tool-local-pusher
       secret:
         secretName: manifest-tool-local-pusher
+- agent: kubernetes
+  cluster: build01
+  cron: '@yearly'
+  decorate: true
+  labels:
+    config-type: modern
+    job-architecture: multi
+    job-env: hypershift-hosted-powervs
+    job-type: launch
+  name: release-openshift-origin-installer-launch-hypershift-hosted-powervs
+  spec:
+    containers:
+    - args:
+      - --gcs-upload-secret=/secrets/gcs/service-account.json
+      - --image-import-pull-secret=/etc/pull-secret/.dockerconfigjson
+      - --lease-server-credentials-file=/etc/boskos/credentials
+      - --report-credentials-file=/etc/report/credentials
+      - --secret-dir=/secrets/ci-pull-credentials
+      - --secret-dir=/usr/local/launch-cluster-profile
+      - --input-hash=$(BUILD_ID)
+      - --input-hash=$(JOB_NAME)
+      - --namespace=$(NAMESPACE)
+      - --target=launch
+      command:
+      - ci-operator
+      env:
+      - name: CLUSTER_DURATION
+        value: "4.14"
+      - name: CLUSTER_TYPE
```

Comment:
Not required it seems, others used as a var for cluster_profile and referred  in UNRESOLVED_CONFIG.

Reply:
ok, removed it.